pkg/controller/servicebindingrequest/olm.go

```diff
@@ -247,29 +245,24 @@ func buildDescriptorsFromAnnotations(annotations map[string]string) (
 
 	acc := make(map[string][]string)
 
-	for n, v := range annotations {
+	for n, v := range in {
 		// Iterate all annotations and compose related Spec and Status descriptors, where those
 		// descriptors should be grouped by field path.	So, for example, the "status.dbCredentials"
 		// field path should accumulate all related annotations, so the StatusDescriptor referring
 		// "status.dbCredentials" have both "user" and "password" XDescriptors.
 
-		// do not process unknown annotations
-		if !strings.HasPrefix(n, ServiceBindingOperatorAnnotationPrefix) {
-			continue
-		}
-
-		// annotationName has the binding information encoded into it.
-		bindingInfo, err := NewBindingInfo(n, v)
+		bindingInfo, err := annotations.NewBindingInfo(n, v)
 		if err != nil {
-			return nil, nil, err
+			// continue to the next annotation if an error is returned
+			continue
```

Comment:
Why do we ignore error here? 
Shouldn't we atleast log the error?

Reply:
We probably can emit k8s events for this in addition to a trace log for development purpose, since this is a user input error as the annotation can't be processed.

Reply:
Did we add debug logging for this?